crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -300,14 +316,8 @@ impl Reader {
                     }
                 }
 
-                Some((celestia_height, blocks)) = block_stream.next() => {
-                    debug!(
```

Comment:
moved the event construction out of this select-loop into the function that actually performs the fetch.

It's better practice to emit events in their correct scopes (instead of glue, like here)